packages/compiler-cli/test/compliance/r3_compiler_compliance_spec.ts

```diff
@@ -675,6 +675,35 @@ describe('compiler compliance', () => {
       expectEmit(source, OtherDirectiveFactory, 'Incorrect OtherDirective.ɵfac');
     });
 
+    it('should convert #myApp selector to ["", "id", "myApp"]', () => {
```

Comment:
Test description should match test

```suggestion
    it('should convert #my-app selector to ["", "id", "my-app"]', () => {
```